packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -214,6 +214,7 @@ export enum TelemetryProperty {
   DebugAppId = "debug-appid",
   DebugProjectComponents = "debug-project-components",
   DebugCheckResults = "debug-check-results",
+  DebugErrorCodes = "error-codes",
```

Comment:
Is it better to use "debug-error-codes"?

Reply:
Thanks. Fixed